test/bmc/bmc_api_test.rb

```diff
@@ -245,7 +245,6 @@ def test_api_uses_default_smart_proxy_logger
     Proxy::BMC::IPMI.any_instance.stubs(:poweron?).returns(true)
     get "/#{host}/chassis/power/on", args
     assert_not_equal 'Rubyipmi', Proxy::BMC::IPMI.logger.progname
-    assert_equal "./logs/test.log", Proxy::BMC::IPMI.logger.instance_variable_get("@logdev").filename
```

Comment:
Yes, the test is actually getting an internal from the logger instance. It is not available anymore. I don't think this is a good test. Anyway, line 247 already test that the instance is not `rubyipmi`, which is the main purpose. The extra test for the internal was useless IMHO.